src/main/views/govukTemplate.njk

```diff
@@ -4,7 +4,9 @@
 <!--[if gt IE 8]><!--><html lang="{{ html_lang|default('en') }}"><!--<![endif]-->
 <head>
   <meta charset="utf-8" />
-  <title>{{ t(title) }}</title>
+  {% if heading %}
```

Comment:
do we need this condition?

Reply:
Your probably right, now that we have a 'headingVisible' condition variable. Will check the templates throughout the application to verify